mozilla/foundation.mozilla.org
# added mozilla says section

Closes #7378 

**Link to sample test pages**:

Test Page with "Mozilla says" set to Unknown, (Sideways): https://foundation-s-7378-mozil-a7zqkp.herokuapp.com/en/privacynotincluded/write-poor-nature-kind-stage-daughter/
Test Page with "Mozilla says" set to Yes, (Thumbs-Up): https://foundation-s-7378-mozil-a7zqkp.herokuapp.com/en/privacynotincluded/entire-economic-course-writer-most-little-notice/
Test Page with "Mozilla says" set to No, (Thumbs-Down): https://foundation-s-7378-mozil-a7zqkp.herokuapp.com/en/privacynotincluded/guess-sit-take-game-instead-store-example-realize/


*Steps to test**:

1. Visit the three links above, each page should have either a thumbs up, down, or sideways.
2. If it appears that they are rendering correctly, testing is complete! 
**Note:** Please note that this section might look a little empty until we add the two other subsections (time researched, people voted section). This, as well as the dividers will be updated/added in future PR's.
